lib/board.py

```diff
@@ -1,9 +1,29 @@
 class Board:
-    board = {1: " ", 2: " ", 3: " ", 4: " ", 5: " ", 6: " ", 7: " ", 8: " ", 9: " "}
+    def __init__(self):
+        self.board = {
+            "1": " ",
+            "2": " ",
+            "3": " ",
+            "4": " ",
+            "5": " ",
+            "6": " ",
+            "7": " ",
+            "8": " ",
+            "9": " ",
+        }
 
-    def display_board():
-        print(Board.board[1] + "  | " + Board.board[2] + " | " + Board.board[3])
+    def display_board(self):
```

Comment:
This board is "hard coded", which works for now, but I wonder if there is a way to programmatically display the board. Just something to consider, no change needed in this PR.

Reply:
I will think about it.